src/ui/common/src/components/integrations/dialogs/IntegrationTextInputField.tsx

```diff
@@ -26,11 +29,21 @@ export const IntegrationTextInputField: React.FC<IntegrationTextFieldProps> = ({
   required,
   placeholder,
   onChange,
-  value,
+  //value,
   type,
   disabled,
   disableReason,
+  autoComplete,
+  name,
 }) => {
+  const { register, errors, formState } = useFormContext();
+
+  console.log('formContext errors: ', errors);
```

Comment:
NOTE: When an error comes back in validation in react-hook-form, it will be a key of the errors object.

e.g. 
errors.foo = "some error object".
TODO: add errors[key] check to IntegrationtextInputField to render errors appropriately.